lib/mayaUsd/resources/scripts/mayaUsdMergeToUsd.py

```diff
@@ -58,14 +58,22 @@ def createMayaReferenceMenuItem(dagPath, precedingItem):
     pulledPath = pathMapper.fromHost(mayaItem.path())
     prim = mayaUsd.ufe.ufePathToPrim(ufe.PathString.string(pulledPath))
 
-    # If the pulled prim isn't a MayaReference, not our responsibility.
-    if prim.GetTypeName() != 'MayaReference':
-        return ''
-
-    cmd = partial(mayaUsdCacheMayaReference.cacheDialog, fullDagPath, prim)
-    cacheLabel = getMayaUsdLibString('kMenuCacheToUsd')
-    return cmds.menuItem(label=cacheLabel, insertAfter=precedingItem, 
-                         image="cache_to_USD.png", command=cmd)
+    # If the pulled prim doesn't exist anymore, we won't delegate the reponsibility to
```

Comment:
This is something we will iterate a bit more with @womanyoyoyo-adsk. As the code is written now, we are relying on the prim type to decide on the menu items to display. It doesn't work for orphaned objects so we may need to change strategy. The priority of this work is undetermined at this point and this is why we left it outside of the scope for this PR and MAYA-121585.